Assets/Scripts/Combat/Attack.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
What are these imports used for here?

Reply:
Generic Library: We used it for making the list of enemies, allies, and sorting who goes first. 
Linq Library: Used for lambda functions and sorting (Link: https://unity3d.college/2017/07/01/linq-unity-developers/). We used this for turn priority and targeting mostly.
Threading: We used it to just sleep the program when selecting an ability and when the enemies attack. This is temporary to just simulate combat (and not make each move instant).

Reply:
We are using these, but not in the file. Removing now!